README.md

```diff
@@ -43,7 +43,7 @@ part is `3DA2BCE3` -- this lets you check it in the
 >
 > ```
 > $ gpg --with-fingerprint signing-keys/tarsnap-signing-key-2016.asc
-pub  4096R/3DA2BCE3 2016-02-18 Tarsnap source code signing key (Colin Percival) <cperciva@tarsnap.com>
+> pub  4096R/3DA2BCE3 2016-02-18 Tarsnap source code signing key (Colin Percival) <cperciva@tarsnap.com>
```

Comment:
Not sure why this changed here?

Reply:
It's an unrelated 2-char fix that I shoved into the same commit. Look at https://github.com/Tarsnap/tarsnap-public-keys
(at the bottom of the page) and you'll see it.

That said, I suppose there's no harm in having an extra commit, so I'll split it off.